jvcl/run/JvJVCLUtils.pas

```diff
@@ -3317,14 +3317,22 @@ function PanelBorder(Panel: TCustomPanel): Integer;
 function Pixels(Control: TControl; APixels: Integer): Integer;
 var
   Form: TForm;
+  MonitorPPI: Integer;
 begin
   Result := APixels;
   if Control is TForm then
     Form := TForm(Control)
   else
     Form := TForm(GetParentForm(Control));
   if Form.Scaled then
-    Result := Result * Form.PixelsPerInch div 96;
+  begin
+  {$ifdef COMPILER24_UP}
```

Comment:
This should really be an `IFDEF RTLXXX_UP` and needs one more level of indentation

Reply:
I have not all versions of Delphi installed here. According to the official documentation Wiki, this function was already available in Delphi 2010. Would this change be ok for you ?

```delphi
function Pixels(Control: TControl; APixels: Integer): Integer;
var
  Form: TForm;
  MonitorPPI: Integer;
begin
  Result := APixels;
  if Control is TForm then
    Form := TForm(Control)
  else
    Form := TForm(GetParentForm(Control));
  if Form.Scaled then
  begin
    {$ifdef RTL210_UP}
    MonitorPPI := Screen.MonitorFromWindow(Form.Handle).PixelsPerInch;
    {$else}
    MonitorPPI := Screen.PixelsPerInch;
    {$endif}
    Result := MulDiv(Result, MonitorPPI, 96);
  end;
end;
```

Reply:
The first one is fine by me